packages/react-bootstrap-table2-paginator/src/const.js

```diff
@@ -1,4 +1,5 @@
 export default {
+  STYLE: {},
```

Comment:
I think it's a little not make sense to define a style object as a constant variable, you can just remove it and defined in the component